src/Util.hs

```diff
@@ -22,11 +24,10 @@ import           Lens.Micro.Platform
 takeNext :: [a] -> [a]
 takeNext = take 1 . drop 1
 
--- | Turn a list of Menu into an `Either NoMenu [Menu]`
-menusToEitherNoLunch :: [Menu] -> Either NoMenu [Menu]
+menusToEitherNoLunch :: [Menu] -> Either NoMenu (NonEmpty Menu)
 menusToEitherNoLunch = \case
   [] -> Left NoLunch
-  xs -> Right xs
+  xs -> (Right . NE.fromList) xs
```

Comment:
This is fine, but I am thinking if one should be explicitly using the constructor for `NonEmpty`... It doesn't really introduce any potential issues doing it this way however

Reply:
Yes :smile_cat: